p2p/dhtdiscovery/discovery.go

```diff
@@ -197,14 +205,15 @@ type Discovery struct {
 	// how often to check if we have enough peers
 	period time.Duration
 	// timeout used for connections
-	timeout             time.Duration
-	bootstrapDuration   time.Duration
-	minPeers, highPeers int
-	backup, bootnodes   []peer.AddrInfo
-	advertiseDelay      time.Duration
-	advertiseInterval   time.Duration
-	advertiseRetryDelay time.Duration
-	findPeersRetryDelay time.Duration
+	timeout                 time.Duration
+	bootstrapDuration       time.Duration
+	minPeers, highPeers     int
+	backup, bootnodes       []peer.AddrInfo
+	advertiseDelay          time.Duration
+	advertiseInterval       time.Duration
+	advertiseIntervalSpread time.Duration
```

Comment:
Instead of making this configurable would it make sense to instead have it a relative amount, e.g. 1%?